### PR TITLE
chore(ec2): missing vpc endpoint per service

### PR DESCRIPTION
**Pull request by watany-dev**
Continued work on https://github.com/aws/aws-cdk/pull/22481/

We are focusing on "multiple endpoints per service".
Some of the property names were chosen arbitrarily, so if you have any comments on them, please let us know.

----

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts [hunk 1]
```diff
@@ -264,34 +264,56 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
   public static readonly ACCOUNT_MANAGEMENT = new InterfaceVpcEndpointAwsService('account');
   public static readonly APP_MESH = new InterfaceVpcEndpointAwsService('appmesh-envoy-management');
   public static readonly APP_RUNNER = new InterfaceVpcEndpointAwsService('apprunner');
+  public static readonly APP_RUNNER_REQUESTS = new InterfaceVpcEndpointAwsService('apprunner.requests');
   public static readonly APPLICATION_MIGRATION_SERVICE = new InterfaceVpcEndpointAwsService('mgn');
+  public static readonly APPSTREAM = new InterfaceVpcEndpointAwsService('appstream.api');
```
Comment (TheRealAmazonKendra) [inline packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts:269]:
```suggestion
  public static readonly APPSTREAM_API = new InterfaceVpcEndpointAwsService('appstream.api');
```
I'm wondering if this should be consistent with what the endpoint actually is.

File: packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts [hunk 2]
```diff
@@ -326,39 +360,86 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
   public static readonly IAM_ROLES_ANYWHERE = new InterfaceVpcEndpointAwsService('rolesanywhere');
   public static readonly INSPECTOR = new InterfaceVpcEndpointAwsService('inspector2');
   public static readonly IOT_CORE = new InterfaceVpcEndpointAwsService('iot.data');
+  public static readonly IOT_LORAWAN = new InterfaceVpcEndpointAwsService('iotwireless.api');
```
Comment (TheRealAmazonKendra) [inline packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts:363]: This one doesn't seem to match up. Is this correct?

**Review by TheRealAmazonKendra — CHANGES_REQUESTED** [review]
Just a couple comments inline.

File: packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts (hunk 2 above)
Comment (watany-dev) [reply packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts:363]: @TheRealAmazonKendra 
I too am struggling with the naming of it. The basis for this decision is twofold.

1. the line of the following page searched for "'iotwireless.api'"
https://docs.aws.amazon.com/ja_jp/vpc/latest/privatelink/integrated-services-vpce-list.html

2. the official documentation from a Google search for "iotwireless.api"
https://docs.aws.amazon.com/ja_jp/iot/latest/developerguide/connect-iot-lorawan-onboard-control-plane-endpoint.html

Comment (TheRealAmazonKendra) [reply packages/@aws-cdk/aws-ec2/lib/vpc-endpoint.ts:363]: Hmm. Maybe this to keep it consistent with documentation?
```suggestion
  public static readonly IOT_CORE_FOR_LORAWAN = new InterfaceVpcEndpointAwsService('iotwireless.api');
```

**Review by TheRealAmazonKendra — DISMISSED** [review]

**Review by corymhall — APPROVED** [review]
@watany-dev this is great, thanks for doing this!

**Pull request merged**